### PR TITLE
Feat: add the bootstrap input style in dashboard documents.

### PR DESCRIPTION
**Pull request by stu01509**
Add the bootstrap input style in dashboard documents.

#### Short description of what this resolves:
- Remove outdated code and inline-style.
- Add the bootstrap input style.

### CODE REVIEW

File: templates/documents/general_list.html [hunk 1]
```diff
@@ -49,41 +49,39 @@
...
-				<legend>{xlt t='Documents List'}</legend>
-                <div style="padding: 0 10px">
-                    <div class="ui-widget form-inline" class='float-right'>
-                         <select id="selectPatient" class="form-control" type="text" data-placeholder="{$place_hld|attr}" style="width:150px;">
-                            <option></option></select>
-                        <button id='pid' class="btn btn-primary pBtn" type="button" class='float-right' style="border-radius: 0 0.25rem 0.25rem 0">0</button>
+                    <legend>{xlt t='Documents List'}</legend>
+                    <div class="pl-3">
+                        <div class="ui-widget form-inline float-right">
+                            <select id="selectPatient" class="form-control" type="text" data-placeholder="{$place_hld|attr}" style="width:150px;">
```
Comment (stu01509) [inline templates/documents/general_list.html:71]: Hi @tywrenn 

Seems like `ui-widget` it is jQuery-UI class, can I remove it?

Comment (tywrenn) [reply templates/documents/general_list.html:71]: Yep - just make sure you find a replacement for it if needed

Comment (stu01509) [reply templates/documents/general_list.html]: OK, update the commit.

File: templates/documents/general_upload.html [hunk 2]
```diff
@@ -11,70 +11,82 @@
...
-        <span>{xlt t="Source File Path"}:</span> <input type="file" name="file[]" id="source-name" multiple="true" />
+    <div class="text">
+        {xlt t="NOTE: Uploading files with duplicate names will cause the files to be automatically renamed (for example, file.jpg will become file.1.jpg). Filenames are considered unique per patient, not per category."}
+        <br/>
+        <br/>
     </div>
-    <div class="form-group">
-        <p>(<small>{xlt t="Select below to Zip Directory of image slices."}</small>)</p>
-        <input type="file" name="dicom_folder[]" id="dicom_folder" multiple directory=""  webkitdirectory="" moxdirectory="" />
+    <div class="text bold">
```
Comment (tywrenn) [inline templates/documents/general_upload.html:29]: Replace the outdated `bold` class with `font-weight-bold`

**Comment by stu01509** [comment]
Update the commit.

File: templates/documents/general_list.html [hunk 3]
```diff
@@ -49,41 +49,39 @@
...
-				<legend>{xlt t='Documents List'}</legend>
-                <div style="padding: 0 10px">
-                    <div class="ui-widget form-inline" class='float-right'>
-                         <select id="selectPatient" class="form-control" type="text" data-placeholder="{$place_hld|attr}" style="width:150px;">
-                            <option></option></select>
-                        <button id='pid' class="btn btn-primary pBtn" type="button" class='float-right' style="border-radius: 0 0.25rem 0.25rem 0">0</button>
+                    <legend>{xlt t='Documents List'}</legend>
+                    <div class="pl-3">
+                        <div class="form-inline float-right" id="patientSearch">
+                            <select id="selectPatient" class="form-control" type="text" data-placeholder="{$place_hld|attr}" style="width:150px;">
```
Comment (tywrenn) [inline templates/documents/general_list.html:71]: This should not have a set inline width

File: templates/documents/general_list.html [hunk 4]
```diff
@@ -49,41 +49,39 @@
...
+                                <option></option>
+                            </select>
+                            <button id='pid' type="button" class='float-right btn btn-primary pBtn' style="border-radius: 0 0.25rem 0.25rem 0">0</button>
```
Comment (tywrenn) [inline templates/documents/general_list.html:74]: Can't believe I caught this one: Replace the 0 with `&times;` for the text inside the button. Also remove the border-radius inline style here.

File: templates/documents/general_upload.html [hunk 5]
```diff
@@ -11,70 +11,82 @@
...
-		<select class="form-control" name='form_filename'>{$TEMPLATES_LIST}</select>
-	</div>
-</div>
+    <input type="hidden" name="csrf_token_form" value="{$CSRF_TOKEN_FORM|attr}" />
+    <input type='hidden' name='patient_id' value='{$patient_id|attr}' />
+    <div class='col-sm-6'>
+        <p class='text font-weight-bold'>{xlt t="Download document template for this patient and visit"}</p>
+        <div class="input-group">
+            <span class="input-group-prepend">
+                <button type="submit" class="btn btn-download">{xlt t="Fetch"}</button>
```
Comment (tywrenn) [inline templates/documents/general_upload.html:85]: This probably needs `btn-primary` tacked on this button...

**Comment by stu01509** [comment]
update the commit.

**Comment by bradymiller** [comment]
hi @stu01509 and @tywrenn , Bringing in. Thanks for the improvements! -brady

**Pull request merged**